src/test/java/uk/gov/beis/els/ControllerSmokeTest.java

```diff
@@ -45,8 +46,10 @@ public void abstractSetUp() {
         .webAppContextSetup(context)
         .build();
 
-    ignoredEndpoints = new ArrayList<>();
-    ignoredEndpoints.add("/swagger-ui.html");
+    ignoredEndpoints = Arrays.asList(
```

Comment:
Nit: worth renaming this to `ignoredEndpointPrefixes` just to make it clear this will match anything that starts with these strings.